examples/llm-api/quickstart_example.py

```diff
@@ -1,4 +1,5 @@
-from tensorrt_llm import LLM, SamplingParams
+from tensorrt_llm import SamplingParams
+from tensorrt_llm._tensorrt_engine import LLM
```

Comment:
The test_pip_install will fail due to the CI docker env if swith to PyT, thus we pause the change.